vignettes/EIRprevmatch.Rmd

```diff
@@ -0,0 +1,161 @@
+---
+title: "Matching PfPR2-10 to EIR"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{EIRprevmatch}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup, message=F, warning=F}
+library(tidyverse)
+library(malariasimulation)
+library(mgcv)
+library(fuzzyjoin)
```

Comment:
Can we remove these dependencies?

It takes a long time to run tests atm and this would take a lot longer
```suggestion
library(malariasimulation)
```